app/views/organisations/show.html.erb

```diff
@@ -72,30 +55,35 @@
   <thead>
     <tr>
       <th>User</th>
-      <th>Roles</th>
-      <th>Applications</th>
+      <th>Organisation Admin</th>
+      <th>Applications and Roles</th>
       <th></th>
     </tr>
   </thead>
 
   <tbody>
     <% if @organisation.memberships.any? %>
-      <%- @organisation.memberships.each do |membership| %>
+      <%- @organisation.memberships.order(:id).each do |membership| %>
         <%= content_tag_for(:tr, membership) do %>
           <td><%= link_to membership.user.name, user_path(membership.user) %></td>
+          <td><%= membership.is_organisation_admin? ? "✓" : "✗" %></td>
```

Comment:
There should be something in the DoorkeeperHelper for this? It certainly uses these characters anyway.